kibana-reports/common/index.ts

```diff
@@ -2,3 +2,7 @@ export const PLUGIN_ID = 'opendistroKibanaReports';
 export const PLUGIN_NAME = 'opendistro_kibana_reports';
 
 export const API_PREFIX = '/api/reporting';
+export const API = {
```

Comment:
May be more descriptive names for constants?

Reply:
Name it `REPORTS_SCHEDULER_API` instead.